PowerShellEditorServices.build.ps1

```diff
@@ -19,19 +19,19 @@ param(
 #Requires -Modules @{ModuleName="InvokeBuild";ModuleVersion="3.2.1"}
 
 $script:IsUnix = $PSVersionTable.PSEdition -and $PSVersionTable.PSEdition -eq "Core" -and !$IsWindows
+$script:IsRosetta = $script:IsUnix -and (sysctl -n sysctl.proc_translated) -eq 1 # Mac M1
```

Comment:
`$IsMacOS` might be more specific here -- otherwise I'd expect this command to fail on Linux

Reply:
```suggestion
$script:IsRosetta = $IsMacOS -and (sysctl -n sysctl.proc_translated) -eq 1 # Mac M1
```

Reply:
Good catch!